torch/_inductor/codegen/rocm/rocm_kernel.py

```diff
@@ -132,7 +133,8 @@ def def_kernel(
                 self.args.output_buffers[node.get_name()] = name
 
         arg_defs, *_ = self.args.cpp_argdefs()
-        return f"PT_EXPORT int {self.kernel_name}({', '.join(arg_defs)}, {self._EXTRA_CPP_ARGS})"
+
+        return f"PT_EXPORT int {self.kernel_name}({', '.join(arg_defs)}, {', '.join(size_args)}, {self._EXTRA_CPP_ARGS})"
```

Comment:
qq: what does `size_args` look like?

Reply:
`size_args` is a list of [M, N, K, LDA, LDB, LDC, LDD]

Reply:
I see, thanks for explaining, so we need scalar here.

Then this makes sense! `size_hint(x)` should produce scalar if `x` is a scalar or backed symint. It won't if `x` is an unbacked symint, but less common.